docs/source/models/causes/neonatal/index.rst

```diff
@@ -199,47 +200,61 @@ unit time.
 Modeling Strategy
 +++++++++++++++++
 
-The neonatal death model requires only the probability of death (aka "mortality risk") for the early and late neonatal time periods.  But computing this for an individual simulant is a bit complicated.  It will follow the pattern from the general mortality component in :code:`vivarium_public_health`, and work in rate space to make the math simpler.  The final step will be converting from rates to risks:
+The neonatal death model requires only the probability of death (aka "mortality risk") for the early and late neonatal time periods. Rather than using GBD mortality rates and converting them into probability of deaths, we will use mortality risk as direct input data into our model. We will calculate mortality risk input data as age-specific death counts divided by live birth counts from GBD.
+
+Note that this strategy does not require any conversion between rates to probabilities NOR does it require any scaling to the duration of the age group. The mortality risk calculated as described below already represents the probability of dying within a neonatal age group and can be used directly as such in the simulation.
+
+To avoid confusion with mortality *rates* (typically referred to as the all-cause mortality rate, ACMR, or cause-specific mortality rates, CSMRs), we will refer to mortality *risk* as ACMRisk (all-cause mortality risk) and CSMRisk (cause-specific mortality risk), where:
 
 .. math::
-    \begin{align*}
-    \text{mr}_i &= 1-\exp(-\text{ACMR}_i \cdot \Delta t),
-    \end{align*}
 
-where :math:`\text{mr}_i` is the probability of mortality for simulant :math:`i` during the early or late neonatal period, :math:`\text{ACMR}_i` is the all-cause mortality rate for the early or late neonatal period, and :math:`\Delta t` is length of that period in years.
+  ACMRisk_{ENN} = \frac{\text{deaths due to all causes in the ENN age group}}{\text{live births}}
+
+  ACMRisk_{LNN} = \frac{\text{deaths due to all causes in the LNN age group}}{\text{live births} - \text{deaths due to all causes in the ENN age group}}
+
+and 
+
+.. math::
+
+  CSMRisk_{ENN} = \frac{\text{deaths due to cause C in the ENN age group}}{\text{live births}}
```

Comment:
The fact that this refers to "cause C" suggests that the "C" in "CSMRisk" is a variable representing an actual cause, e.g., for sepsis we might write "sepsisSMRisk". Is that the intention? Or is the "C" in CSMR still just an abbreviation for the word "cause"? In which case maybe in line 215 it would be good to define the C used here, e.g., change "and" to "and for a given cause C," or something.